pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -42,6 +43,10 @@ type podCounter interface {
 	ReadyCount() (int, error)
 }
 
+type delayer interface {
+	Delay(now time.Time, value int32) int32
```

Comment:
```suggestion
	delay(now time.Time, value int32) int32
```
or we expect external impls?